lib/call.js

```diff
@@ -369,6 +413,120 @@ var Call = function (client) {
 		})
 		.asCallback(callback);
 	};
+
+	/**
+	 * Get events for the call.
+	 * @param {String} callId The ID of the call
+	 * @param {Function} [callback] Callback with the event list
+	 * @return {Array.<EventResponse>} A promise for the event list
+	 * @example
+	 * // Promise
+	 * client.Call.getEvents(callId).then(function (events) {});
+	 * // Callback
+	 * client.Call.getEvents(callId, function (err, events) {});
+	 */
+	this.getEvents = function (callId, callback) {
+		return client.makeRequest({
+			path   : "calls/" + callId + "/events",
+			method : "GET"
+		})
+		.then(function (response) {
+			return response.body;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Get a single event for the call.
+	 * @param {String} callId The ID of the call
+	 * @param {String} eventId The ID of the event to get
+	 * @param {Function} [callback] Callback with the event list
+	 * @return {EventResponse} A promise for the event list
+	 * @example
+	 * // Promise
+	 * client.Call.getEvent(callId, evenId).then(function (callEvent) {});
+	 * // Callback
+	 * client.Call.getEvent(callId, eventId, function (err, callEvent) {});
+	 */
+	this.getEvent = function (callId, eventId, callback) {
+		return client.makeRequest({
+			path   : "calls/" + callId + "/events/" + eventId,
+			method : "GET"
+		})
+		.then(function (response) {
+			return response.body;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Get recordings for the call.
+	 * @param {String} callId The ID of the call
+	 * @param {Function} [callback] Callback with the recording list
+	 * @return {Array.<RecordingResponse>} A promise for the recording list
+	 * @example
+	 * // Promise
+	 * client.Call.getRecordings(callId).then(function (list) {});
+	 * // Callback
+	 * client.Call.getRecordings(callId, function (err, list) {});
+	 */
+	this.getRecordings = function (callId, callback) {
+		return client.makeRequest({
+			path   : "calls/" + callId + "/recordings",
+			method : "GET"
+		})
+		.then(function (response) {
+			return response.body;
+		})
+		.asCallback(callback);
+	};
+
+	/**
+	 * Get transcriptions for the call.
+	 * @param {String} callId The ID of the call
+	 * @param {Function} [callback] Callback with the transcription list
+	 * @return {Array.<RecordingResponse>} A promise for the transcription list
+	 * @example
+	 * // Promise
+	 * client.Call.getTranscriptions(callId).then(function (list) {});
+	 * // Callback
+	 * client.Call.getTranscriptions(callId, function (err, list) {});
+	 */
+	this.getTranscriptions = function (callId, callback) {
```

Comment:
Missing `transcriptionResponse` in docs... not sure if it makes sense to link to [transcriptons Response](https://github.com/bandwidthcom/node-bandwidth/blob/2.0-preview/lib/recording.js#L132) in recording. Are the payloads the same?